src/pages/_document.tsx

```diff
@@ -7,6 +7,10 @@ export default function Document() {
         <link rel="manifest" href="/manifest.json" />
         <link rel="apple-touch-icon" href="/apple-touch-icon.png" />
         <meta name="theme-color" content="#fff" />
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0"
+        />
```

Comment:
This currently prevents mobile users from zooming in, which is pretty bad for accessibility. You only need `initial-scale`.

```suggestion
        <meta
          name="viewport"
          content="width=device-width, initial-scale=1.0"
        />
```

If the mobile browser is zooming the page out regardless, then there's probably some horizontal scroll due to an element that is too wide, which should be fixed instead of disabling zoom

Reply:
Just going to remove this tag since I don't think it really accomplished anything functionally